modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -194,6 +195,8 @@ export default class NestedTabs extends Base {
 			() => this.onShowTabContent( $requestedContent ),
 		);
 		$requestedContent.removeAttr( 'hidden' );
+
+		this.handleElementHandlers();
 	}
 
 	onShowTabContent( $requestedContent ) {
```

Comment:
@hein-obox I found this issue can be fixed with a one liner 😄 

In `nested-tabs.js` add `elementorFrontend.elements.$window.trigger( 'resize' );` into `onShowTabContent()`

```
onShowTabContent( $requestedContent ) {
        elementorFrontend.elements.$window.trigger( 'elementor-pro/motion-fx/recalc' );
        elementorFrontend.elements.$window.trigger( 'elementor/nested-tabs/activate', $requestedContent );
        elementorFrontend.elements.$window.trigger( 'resize' );
}
```

I got the inspiration from here:
https://github.com/elementor/elementor-pro/blob/main/modules/video-playlist/assets/js/frontend/base-tabs.js#L137

Please check it and let me know. Thanks!

Reply:
@louiswol94 

Very interesting!

I am hesitant to implement it though, for 2 reasons: 
1. This would trigger all resize events on the page. I fear that will trigger many unneccessary events.
2. The background video handler by default will resize all videos when it gets triggered. I have made our change event call conditional. In my code only one specific video will be resized.

I have removed the run event handler function (as per your example) and trigger the background resize from the nested tabs immediately.

Reply:
@hein-obox Not sure I agree. If this is the solution in the base tabs, then it should be safe for us to use, it was just lost because we overwrote the function.

Reply:
@hein-obox I see we did not extend it yes ;-) But the fact that this is used in the base tabs means that it should be a viable solution. Please implement it 🙏 It will make this PR a lot less complex. Thanks.

Reply:
@louiswol94 Implemented. Advantage: we are 100% that this solution works, as this solution has been used for a long time already.